### DISCUSSION TITLE
6月15日支付年费，邮箱未收到激活码，请解决或退款

### DISCUSSION THREAD

**Discussion by HawkYin** [getlantern/lantern-forum-cn#616] [General]
微信支付，交易单号：4200002146202406155856947955
邮箱：hawk_yinjun@qq.com

**Comment by MaxMKMax** [comment]
@HawkYin 好的. 我帮您提交问题. 请您过几天查看帖子或邮箱.

**Comment by warm3snow** [comment]
问一下修复好了没？或者退款了没，碰到相同的问题。

**Reply by MaxMKMax** [reply]
@warm3snow 
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
您好. 为了保护您的隐私. 请您发支付单据和账号邮箱到这个链接. 谢谢.

**Comment by MaxMKMax** [comment]
@HawkYin 负责人说您直接登陆就可以了. 您是因为没登录. 您的账号没问题呢.

**Comment by HawkYin** [comment]
还未处理



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年6月20日(周四) 下午3:17
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 6月15日支付年费，邮箱未收到激活码，请解决或退款 (Discussion #616)




 
问一下修复好了没？或者退款了没，碰到相同的问题。

**Comment by HawkYin** [comment]
我没收到激活码



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年6月21日(周五) 凌晨3:02
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 6月15日支付年费，邮箱未收到激活码，请解决或退款 (Discussion #616)




 
@HawkYin 负责人说您直接登陆就可以了. 您是因为没登录. 您的账号没问题呢.

**Comment by JerryMouse2022** [comment]
@Mike777Khoo 都好几天了，也没回应
https://github.com/getlantern/lantern-forum-cn/discussions/406?sort=new#discussioncomment-9367985
